packages/api/db/migration/20230831015743_create_revenue_type_table.js

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (c) 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = async function (knex) {
+  await knex.schema.createTable('revenue_type', (table) => {
+    table.increments('revenue_type_id').primary();
+    table.string('revenue_name').notNullable();
+    table.uuid('farm_id').references('farm_id').inTable('farm').defaultTo(null);
+    table.boolean('deleted').notNullable().defaultTo(false);
+    table.string('created_by_user_id').references('user_id').inTable('users');
+    table.string('updated_by_user_id').references('user_id').inTable('users');
+    table.dateTime('created_at').notNullable();
+    table.dateTime('updated_at').notNullable();
+    table.string('revenue_translation_key').notNullable();
+  });
+
+  await knex.schema.alterTable('sale', (table) => {
+    table.integer('revenue_type_id').references('revenue_type_id').inTable('revenue_type');
+  });
+
+  // Prepopulate with one revenue type (crop_sale)
+  await knex('revenue_type').insert({
+    revenue_type_id: 1,
+    revenue_name: 'crop_sale',
```

Comment:
I think this should be plain english ? 'Crop sale' or something without snake case? I am seeing it I think in both task_type and farmExpenseType.

Reply:
It should indeed!